spec/features/favourites_feature_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+feature 'favouriting photos' do
+
+  scenario 'a user can favourite a photo, which updates the photo favourite count', js: true do
+    sign_up_with("test@example.com")
+    user = User.first
+    photo = Photo.create(caption: "test", image: File.new('./spec/assets/images/burger-king.png'), user: user)
```

Comment:
Metrics/LineLength: Line is too long. [110/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)